packages/account-v2/src/pages/ConnectedApps/ConnectedAppsTable.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { OauthApps } from '@deriv/api-types';
+import { Button, Table } from '@deriv-com/ui';
+import { connectedAppsHeader, connectedAppsHeaderOrder, getConnectedAppsScopes } from '../../utils/connectedAppsUtils';
+
+type TConnectedAppsTable = {
+    connectedApps: OauthApps;
+    // handleToggleModal: (app_id: number) => void;
+};
+
+export const ConnectedAppsTable = ({ connectedApps }: TConnectedAppsTable) => {
+    const connectedAppsColumns = connectedAppsHeaderOrder.map(header_name => ({
+        header: connectedAppsHeader[header_name],
+    }));
+
+    const connectedAppsRows = connectedApps.map(connectedApp => ({
+        lastLogin: connectedApp?.last_used,
+        name: connectedApp.name,
+        permission: getConnectedAppsScopes(connectedApp.scopes),
+    }));
+
+    return (
+        <Table
+            columns={connectedAppsColumns}
+            data={connectedAppsRows}
+            isFetching={false}
+            // eslint-disable-next-line @typescript-eslint/no-empty-function
+            loadMoreFunction={() => {}}
+            renderHeader={header => <span>{header}</span>}
+            rowRender={data => (
+                <div className='grid grid-flow-col text-default'>
```

Comment:
Can these div's be changed to <tr> tags? Its more semantic that way and will help u pick elements based on role while testing

Reply:
What kind of tag do you suggest? Since, its just text we are displaying using `<span>`made sense 😄

Reply:
Github has omited the tag. I meant `<tr>`

Reply:
It makes sense to use `<tr> `inside `table` tag but since, deriv-ui is rendering the rows inside `<div>`, it throws error saying
`Warning: validateDOMNesting(...): <tr> cannot appear as a child of <div>.`
![Screenshot 2024-03-19 at 2 48 31 PM](https://github.com/binary-com/deriv-app/assets/125863995/fd20bd6a-36af-40c4-a488-075e1458785d)